python/pipeline/experiment.py

```diff
@@ -593,4 +593,35 @@ class AutoProcessing(dj.Manual):
     autosegment=false   :boolean       # segment somas in the first channel with default method
     """
 
+@schema
+class Projector(dj.Lookup):
+    definition = """
+    # projector specifications
+    projector_id        : tinyint                               # projector id
+    refresh_rate        : tinyint                               # refresh rate in Hz
+    ---
+    red                 : enum('UV', 'Green', 'Blue', 'None')   # color to be used for red channel
+    green               : enum('UV', 'Green', 'Blue', 'None')   # color to be used for green channel
+    blue                : enum('UV', 'Green', 'Blue', 'None')   # color to be used for blue channel
+    pixel_width         : smallint                              # number of pixels in width
+    pixel_height        : smallint                              # number of pixels in height
+    """
+    contents = [
+        [0, 60, 'UV', 'Green', 'Blue', 1920, 1080],
+        [1, 60, 'UV', 'UV', 'Green', 1140, 912],
+        [1, 30, 'UV', 'UV', 'Green', 1140, 912]
+    ]
+
+@schema
+class ProjectorDisplay(dj.Manual):
+    definition = """
+    # projected display specification
+    -> Projector
+    -> Rig
+    target_distance     : decimal(6,3)  # distance from mouse to the projected display in cm
```

Comment:
you sure target distance should be in the primary key

Reply:
yes, this is a primary key because the distance between the mouse and the display later is used for grating generation process https://github.com/DonnieKim411/stimuli/blob/master/matlab/%2Bstimulus/ColorGrate.m#L69

depending on the target_distance, the way grating is visualized on display changes, hence it is a primary key

Reply:
i don't understand what this table is supposed to contain. I don't see how the rig matters in defining the size of displayed width for instance, if a prohjector witht he same config equally far is in rig 1 or 2 that shouldn't matter. Also if target distance matters for creating the grating it should be part of some GratingConfigs table (or is this the GratingConfig table but with a diff name?)

Reply:
in a less technichal note I doubt very much that experimetners will record how far the mouse is to the screen for each scan, nor will it be very exact, chances are this should just be assumed in the condition

Reply:
`target_distance` is not a primary attribute, should be moved to secondary

Reply:
Can multiple projectors be used on the same rig? If not, then we don't need both in the primary key.